core/pom.xml

```diff
@@ -86,6 +86,7 @@ limitations under the License.
     <dependency>
       <groupId>com.google.code.findbugs</groupId>
       <artifactId>jsr305</artifactId>
+      <optional>true</optional>
```

Comment:
Why not 'provided' ?

Reply:
Provided means "we expect the API be provided by container", while optional=true means we don't really need that API